src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateMethod.cs

```diff
@@ -192,6 +191,12 @@ public override void ExecuteCmdlet()
         [ValidateNotNullOrEmpty]
         public bool? EnableAutomaticUpdate { get; set; }
 
+        [Parameter(
+            ParameterSetName = "DefaultParameter",
+            Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public string[] IdentityId { get; set; }
```

Comment:
This needs to go through review.  What are the possible values here, and how can we lead the user into the correct one?

Reply:
IdentityId parameter takes an array of strings (ids), each of which is a form of "/subscriptions/<sub_id> /resourceGroups/<rg_name>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<identity_name>".

User should create this ID using ManagedIdentity API (or cmdlet)

Reply:
Does this not need an IdentityType? The same parameter set issues shoudl apply to both.

Reply:
This needs IdentityType with UserIdentity or SystemIdentityUserIdentity, but a user may not give any Identity at all.   So both are optional.